composer.json

```diff
@@ -47,6 +47,10 @@
 	"replace": {
 		"mediawiki/semantic-mediawiki": "*"
 	},
+	"suggest": {
```

Comment:
These two extensions are the most commonly used ones alongside SMW. I've not added things such as Semantic Maps, because they are rather specific. There are ones for which I'm really not sure, such as SESP, which while general, is still not something used commonly. If you think something else should be added, go ahead.